src/io/vss_store.rs

```diff
@@ -60,7 +64,21 @@ impl VssStore {
 	) -> Self {
 		let inner = Arc::new(VssStoreInner::new(base_url, store_id, vss_seed, header_provider));
 		let next_version = AtomicU64::new(1);
-		Self { inner, next_version, runtime }
+		let internal_runtime = Some(
```

Comment:
Add the @TheBlueMatt analysis here as a comment to explain why we need this (for now)?

>there's a blocking task waiting on the VSS store to finish, but ultimately what's blocking the reactor is another task calling PeerManager::process_pending_events which calls ChannelManager::get_and_clear_pending_msg_events which then tries to take some locks in channelmanager that are held by the thread which is currently blocked waiting on the VSS store.